packages/psss-server/src/routes/country/FetchCountries.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { getSortByKey } from '@tupaia/utils';
+import { PSSS_PERMISSION_GROUP } from '../../constants';
+import { Route } from '../Route';
+
+export class FetchCountries extends Route {
+  async buildResponse() {
+    const countries = await this.entityConnection.fetchCountries();
+    const { accessPolicy } = await this.getSession();
+    const allowedEntities = accessPolicy.getEntitiesAllowed(PSSS_PERMISSION_GROUP);
+
+    const countryData = countries
+      .filter(c => allowedEntities.includes(c.code))
+      .sort(getSortByKey('name'));
```

Comment:
Perhaps `entity-server` could support ordering, in the meantime I believe it's fine to do so here

Reply:
I think @tcaiger created a ticket for us to add this functionality: https://app.zenhub.com/workspaces/active-sprints-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/2658